conf/config.local.php

```diff
@@ -1,9 +1,886 @@
+ * (utilisé seulement si le code OTP fourni est faux)
+ *
+ * Désactiver (false) si vous êtes sûr que votre serveur est toujours à l'heure.
+ *
+ * Défaut : fr.pool.ntp.org
+ */
+
+//const NTP_SERVER = 'fr.pool.ntp.org';
+
+/**
+ * Désactiver l'envoi d'e-mails
+ *
+ * Si positionné à TRUE, l'envoi d'e-mail ne sera pas proposé, et il ne sera
+ * pas non plus possible de récupérer un mot de passe perdu.
+ * Les parties de l'interface relatives à l'envoi d'e-mail seront cachées.
+ *
+ * Ce réglage est utilisé pour la version autonome sous Windows, car Windows
+ * ne permet pas l'envoi d'e-mails.
+ *
+ * Défaut : false
+ * @var bool
+ */
+
+//const DISABLE_EMAIL = false;
+
+
+/**
+ * Hôte du serveur SMTP, mettre à false (défaut) pour utiliser la fonction
+ * mail() de PHP
+ *
+ * Défaut : false
+ */
+
+const SMTP_HOST = true;
+
+/**
+ * Port du serveur SMTP
+ *
+ * 25 = port standard pour connexion non chiffrée (465 pour Gmail)
+ * 587 = port standard pour connexion SSL
+ *
+ * Défaut : 587
+ */
+
+const SMTP_PORT = 25;
+
+/**
+ * Login utilisateur pour le server SMTP
+ *
+ * mettre à null pour utiliser un serveur local ou anonyme
+ *
+ * Défaut : null
+ */
+
+const SMTP_USER = '__APP__';
+
+/**
+ * Mot de passe pour le serveur SMTP
+ *
+ * mettre à null pour utiliser un serveur local ou anonyme
+ *
+ * Défaut : null
+ */
+
+const SMTP_PASSWORD = '__MAIL_PWD__';
+
+/**
+ * Sécurité du serveur SMTP
+ *
+ * NONE = pas de chiffrement
+ * SSL = connexion SSL native
+ * TLS = connexion TLS native (le plus sécurisé)
+ * STARTTLS = utilisation de STARTTLS (moyennement sécurisé)
+ *
+ * Défaut : STARTTLS
+ */
+
+const SMTP_SECURITY = 'NONE';
+
+/**
+ * Nom du serveur utilisé dans le HELO SMTP
+ *
+ * Si NULL, alors le nom renseigné comme SERVER_NAME (premier nom du virtual host Apache)
+ * sera utilisé.
+ *
+ * Defaut : NULL
+ *
+ * @var null|string
+ */
+
+const SMTP_HELO_HOSTNAME = 'localhost';
+
+/**
+ * Adresse e-mail destinée à recevoir les erreurs de mail
+ * (adresses invalides etc.) — Return-Path
+ *
+ * Si laissé NULL, alors l'adresse e-mail de l'association sera utilisée.
+ * En cas d'hébergement de plusieurs associations, il est conseillé
+ * d'utiliser une adresse par association.
+ *
+ * Voir la documentation de configuration sur des exemples de scripts
+ * permettant de traiter les mails reçus à cette adresse.
+ *
+ * Défaut : null
+ */
+
+const MAIL_RETURN_PATH = 'returns@__DOMAIN__';
+
+
+/**
+ * Adresse e-mail expéditrice des messages (Sender)
+ *
+ * Si vous envoyez des mails pour plusieurs associations, il est souhaitable
+ * de forcer l'adresse d'expéditeur des messages pour passer les règles SPF et DKIM.
+ *
+ * Dans ce cas l'adresse de l'association sera indiquée en "Reply-To", et
+ * l'adresse contenue dans MAIL_SENDER sera dans le From.
+ *
+ * Si laissé NULL, c'est l'adresse de l'association indiquée dans la configuration
+ * qui sera utilisée.
+ *
+ * Défaut : null
+ */
+
+const MAIL_SENDER = 'associations@__DOMAIN__';
```

Comment:
Docs are in French but if this is 'from' address then it need to match `__APP__@__DOMAIN__`, otherwise `dovecot` will reject send attempt.